ldi-core/sparql-construct/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/sparqlFunctions/FirstCoordinate.java

```diff
@@ -0,0 +1,27 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.sparqlFunctions;
+
+import org.apache.jena.geosparql.implementation.GeometryWrapper;
+import org.apache.jena.geosparql.implementation.datatype.WKTDatatype;
+import org.apache.jena.sparql.expr.NodeValue;
+import org.apache.jena.sparql.function.FunctionBase2;
+
+import static be.vlaanderen.informatievlaanderen.ldes.ldi.utils.SparqlFunctionsUtils.getNodeValue;
+
+public class FirstCoordinate extends FunctionBase2 {
+
+	public static final String name = "https://w3id.org/tree#firstCoordinate";
```

Comment:
For "public static final" variables we usually use uppercase names: name -> NAME